src/Main.ts

```diff
@@ -78,35 +79,27 @@ window.tsParticles = new Main();
 Object.freeze(window.tsParticles);
 
 /* particles.js compatibility */
-window.particlesJS = (tagId: string, params: IOptions) => {
-    if (console) {
-        console.info("this method is obsolete, please use the new tsParticles.load");
-    }
-
-    return window.tsParticles.load(tagId, params);
-};
-
-window.particlesJS.load = (tagId: string, pathConfigJson: string, callback: (container: Container | undefined) => void) => {
-    if (console) {
-        console.info("this method is obsolete, please use the new tsParticles.loadJSON");
-    }
-
-    window.tsParticles.loadJSON(tagId, pathConfigJson).then(callback).catch((error) => {
-        console.error(error);
-    });
-};
-
-window.particlesJS.setOnClickHandler = (callback: EventListenerOrEventListenerObject) => {
-    if (console) {
-        console.info("this method is obsolete, please use the new tsParticles.setOnClickHandler");
-    }
-
-    window.tsParticles.setOnClickHandler(callback);
-};
-
+/*
+ * @deprecated this method is obsolete, please use the new tsParticles.loadJSON
+ */
+window.particlesJS = (tagId: string, params: IOptions) => ParticlesJS.load(tagId, params);
+
+/*
+ * @deprecated this method is obsolete, please use the new tsParticles.loadJSON
+ */
+window.particlesJS.load = (tagId: string, pathConfigJson: string, callback: (container: Container) => void) => ParticlesJS.loadJson(tagId, pathConfigJson, callback);
+
+/*
+ * @deprecated this method is obsolete, please use the new tsParticles.setOnClickHandler
+ */
+window.particlesJS.setOnClickHandler = (callback: EventListenerOrEventListenerObject) => ParticlesJS.setOnClickHandler(callback);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5052388)